storage/migrate.go

```diff
@@ -89,6 +89,12 @@ func (d *Database) MigrateSchemaTo(ctx context.Context, target int) error {
 		return xerrors.Errorf("check migration sequence: %w", err)
 	}
 
+	// CREATE EXTENSION requires a global lock so it cannot be run inside
```

Comment:
Let's do this after we have obtained the exclusive lock below. The lock is advisory and simply used to ensure that there is only one copy of visor using the database. It won't actually cause postgres to lock anything. I believe the restriction on the extension is that it can't be run inside a transaction.